backend/.gitignore

```diff
@@ -0,0 +1,5 @@
+htmlcov/
```

Comment:
do we need this ignore file here ? I think the idea was to have one project.gitignore to avoid problems. Please check if it is possible to move this to the project .gitignore file

Reply:
everything is now in one gitignore